itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/QuarkusIntegrationTests.java

```diff
@@ -33,12 +34,13 @@
 @RunWith(RedDeerSuite.class)
 @SuiteClasses({
 	PerspectiveTest.class,
-	ApplicationPropertiesContentAssistTest.class,
-	ApplicationPropertiesNewExtensionContentAssistTest.class,
 	CreateNewProjectTest.class,
+	RunProjectWithDebugTest.class,
+	ApplicationPropertiesContentAssistTest.class,
+	InstallQuarkusExtensionTest.class,
 	CreateNewQuarkusConfigurationMavenTest.class,
 	CreateNewQuarkusConfigurationGradleTest.class,
-	InstallQuarkusExtensionTest.class,
+	ApplicationPropertiesNewExtensionContentAssistTest.class,
 	
```

Comment:
You can change ordering of test classes: from easier use cases to more advanced use cases(from perspective test, creating project to debug test).(We use this in different itests suites)